components/aws/sagemaker/tests/integration_tests/scripts/run_integration_tests

```diff
@@ -164,6 +168,43 @@ function install_kfp() {
   echo "[Installing KFP] Pipeline pods are ready"
 }
 
+function install_ack(){
+
+  eksctl utils associate-iam-oidc-provider --cluster ${EKS_CLUSTER_NAME} \
```

Comment:
this is already done in `generate_iam_role`
https://github.com/kubeflow/pipelines/blob/master/components/aws/sagemaker/tests/integration_tests/scripts/generate_iam_role#L85